testsuite/features/step_definitions/common_steps.rb

```diff
@@ -824,9 +824,10 @@
     node.run('yum install wget', true, 600, 'root')
   end
   command1 = "wget --no-check-certificate -O /usr/share/rhn/RHN-ORG-TRUSTED-SSL-CERT http://#{$server.ip}/pub/RHN-ORG-TRUSTED-SSL-CERT"
-  puts node.run(command1, true, 500, 'root').to_s
+  # Replace unicode quotes in the output by single quotes (otherwise, they might break some Cucumber formatters).
+  puts node.run(command1, true, 500, 'root').to_s.gsub(/\\xE2\\x80\\x98/, "'").gsub(/\\xE2\\x80\\x99/, "'")
```

Comment:
I wonder if there is a better way to cover all the cases.
What about the method `dump`? 
https://ruby-doc.org/core-2.7.2/String.html#method-i-dump

I must say I never tried it.

Reply:
Yes, let's do it in that way.